server/src/graphql/typedefs/search.graphql

```diff
@@ -0,0 +1,13 @@
+union SearchItem = Pathway | Step
```

Comment:
Shouldn't this include `Content` as well. After all `Pathway` and `Step` are just names.

Reply:
Yes, this PR is a WIP. I’ll add search for content too.